### PR TITLE
Pending TODO Script Checker in Codebase

### PR DESCRIPTION
**Pull request by zachzundel**
Part of #497 

---

# Description
In this PR is the script designed to check whether or not all TODOs in the repository are properly formatted, and if not, display who needs to update their code.

### CODE REVIEW

**Comment by zachzundel** [comment]
Good catch @terenc3t, I've changed the script.

**Comment by nisdas** [comment]
The build should fail if there are incorrectly formatted todos , but instead it is passing.

**Comment by zachzundel** [comment]
@nisdas -- I'm not sure what you mean, I see the Travis CI as failing.

**Comment by nisdas** [comment]
@zachzundel  The todo test should be failing https://travis-ci.org/prysmaticlabs/prysm/jobs/430378335

**Comment by prestonvanloon** [comment]
+1 The test isn't failing. Travis reports:
```
The command "./scripts/check-todo.sh" exited with 0.
```

**Comment by prestonvanloon** [comment]
Now we just need to ping people to resolve their violating TODOs 😄

**Comment by prestonvanloon** [comment]
All TODOs should be resolved in the correct format! 

Will merge this after the current travis run passes.

**Comment by zachzundel** [comment]
Sweet! Glad to help out.

**Comment by zachzundel** [comment]
Oh, looks like there's an issue with the script. Sorry. Let me figure it out.

File: .travis.yml [hunk 1]
```diff
@@ -10,6 +10,11 @@ matrix:
       script:
         - 
           go get github.com/alecthomas/gometalinter && gometalinter --install && gometalinter ./... --deadline=10m --exclude=client/internal/client_helper.go
+    - os: linux
+      env: 
+        - todos
+      script:
```
[the remark this answers is not in the document]
Comment (prestonvanloon) [reply .travis.yml]: It's basically a no-op. I know it's not intuitive but this is the way it is.

File: .travis.yml [hunk 2]
```diff
@@ -10,6 +10,12 @@ matrix:
       script:
         - 
           go get github.com/alecthomas/gometalinter && gometalinter --install && gometalinter ./... --deadline=10m --exclude=client/internal/client_helper.go
+    - os: linux
+      env: 
+        - todos
+        - install: true # Skip installing go packages
```
Comment (prestonvanloon) [inline .travis.yml]: No, this needs to be at the same level as script.

```yaml
env:
  - todos
install: true # Skip installing go packages
script:
...
```

Comment (zachzundel) [reply .travis.yml]: Sorry -- fixed

**Pull request merged**